### PR TITLE
Add tables and fields needed for Swagger API

### PR DESCRIPTION
**Pull request by colindavey**
Added support for city table and catSubcat table. 
Added fields to the Help Service table. 
Tested with the server running locally.

### CODE REVIEW

File: LittleHelpBook/Server/Services/AirTableService.cs [hunk 1]
```diff
@@ -105,6 +131,24 @@ public async Task<IEnumerable<Place>> GetPlacesPopulatedAsync()
 
                     place.Subcategories = null;
                 }
+                if (place.CatSubcats != null)
+                {
+                    foreach (var id in place.CatSubcats)
+                    {
+                        place.CatSubcatList.Add(await GetCatSubcatAsync(id));
```
Comment (markdav-is) [inline LittleHelpBook/Server/Services/AirTableService.cs:138]: Places allready have categories and categories have subcategories, so this is redundant.  We can leave it for now, but it blots the data.  I'll make a seperate issues to remote it.

**Review by markdav-is — CHANGES_REQUESTED** [review]
please trim this this back to the commits to add the City and I'll approve it.  more smaller commits is better.

**Comment by colindavey** [comment]
The front end needs it all to work with the API.

**Comment by markdav-is** [comment]
nice work!

**Pull request merged**